package.json

```diff
@@ -87,7 +97,11 @@
 		"eslint-plugin-react": "7.7.0",
 		"espree": "3.5.4",
 		"extract-text-webpack-plugin": "4.0.0-beta.0",
+		"glob": "7.1.2",
+		"lerna": "3.0.0-beta.21",
```

Comment:
@nerrad - I'm wondering if this can now safely be moved to devDependencies since it no longer enforces `postinstall` step as noted in #6718.

Reply:
Ya with there being no `postinstall` it would be better to move it.